unit_tests/fields/multi_field_test.cpp

```diff
@@ -44,43 +44,43 @@ FLOW123D_FORCE_LINK_IN_PARENT(field_constant)
 FLOW123D_FORCE_LINK_IN_PARENT(field_formula)
 FLOW123D_FORCE_LINK_IN_PARENT(field_fe)
 
-string field_constant_input = R"YAML(
-common: !FieldConstant 
-  value:
-   - 1
-   - 2
-   - 3
-transposed:
- - !FieldConstant
-   value: 1
- - !FieldConstant
-   value: 2
- - !FieldConstant
-   value: 3
-)YAML";
-
-TEST(MultiField, transposition) {
-	MultiField<3, FieldValue<3>::Scalar> empty_mf;
-	Input::Type::Record in_rec = Input::Type::Record("MultiFieldTest","")
-	    .declare_key("common", empty_mf.get_multifield_input_type(), Input::Type::Default::obligatory(),"" )
-	    .declare_key("transposed", empty_mf.get_multifield_input_type(), Input::Type::Default::obligatory(),"" )
-	    .close();
-
-	Input::ReaderToStorage json_reader(field_constant_input, in_rec, Input::FileFormat::format_YAML);
-	Input::Record input = json_reader.get_root_interface<Input::Record>();
-
-	Input::Array common = input.val<Input::Array>("common");
-	Input::Array transposed = input.val<Input::Array>("transposed");
-
-	EXPECT_EQ(common.size(), transposed.size());
-
-	auto it_c = common.begin<Input::AbstractRecord>();
-	for (auto it_t = transposed.begin<Input::AbstractRecord>(); it_t != transposed.end(); ++it_t, ++it_c) {
-		Input::Record rec_t = (*it_t);
-		Input::Record rec_c = (*it_c);
-		EXPECT_DOUBLE_EQ( rec_t.val<double>("value"), rec_c.val<double>("value") );
-	}
-}
+//string field_constant_input = R"YAML(
+//common: !FieldConstant
+//  value:
```

Comment:
Try `common: [1, 2, 3]` as the input to the auto conversion. Try to disable transposition support for the MultiFields.